docs/data/material/getting-started/templates/checkout/Checkout.tsx

```diff
@@ -55,68 +116,276 @@ export default function Checkout() {
+              href="/material-ui/getting-started/templates/landing-page/"
+              sx={{ alignSelf: 'start', ml: '-8px' }}
+            >
+              Back to
+              <img
+                src={
+                  'https://assets-global.website-files.com/61ed56ae9da9fd7e0ef0a967/61f12e6faf73568658154dae_SitemarkDefault.svg'
+                }
+                style={logoStyle}
+                alt="logo of sitemark"
+              />
+            </Button>
+            <Info />
+          </Box>
+        </Grid>
+        <Grid
+          item
+          sm={12}
+          md={7}
+          lg={8}
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            maxWidth: '100%',
+            width: '100%',
+            backgroundColor: { xs: 'transparent', sm: 'background.default' },
+            alignItems: 'start',
+            pt: { xs: 2, sm: 4 },
+            px: { xs: 2, sm: 10 },
+            gap: { xs: 4, md: 8 },
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: { sm: 'space-between', md: 'flex-end' },
+              alignItems: 'center',
+              width: '100%',
+              maxWidth: { sm: '100%', md: 600 },
+              height: 40,
+            }}
+          >
+            <Box
+              sx={{
+                display: { sm: 'flex', md: 'none' },
+                flexDirection: 'row',
+                width: '100%',
+                alignItems: 'center',
+              }}
+            >
+              <Button
+                startIcon={<ArrowBackRoundedIcon />}
+                component="a"
+                href="/material-ui/getting-started/templates/landing-page/"
+                sx={{ alignSelf: 'start' }}
+              >
+                Back to
+                <img
+                  src={
+                    'https://assets-global.website-files.com/61ed56ae9da9fd7e0ef0a967/61f12e6faf73568658154dae_SitemarkDefault.svg'
+                  }
+                  style={logoStyle}
+                  alt="logo of sitemark"
+                />
+              </Button>
+            </Box>
+            <ToggleColorMode
+              mode={mode}
+              toggleColorMode={toggleColorMode}
+              showCustomTheme={showCustomTheme}
+            />
+          </Box>
+          <Card
+            sx={{
+              display: { xs: 'flex', md: 'none' },
+              width: '100%',
+            }}
+          >
+            <CardContent
+              sx={{
+                display: 'flex',
+                width: '100%',
+                alignItems: 'center',
+                ':last-child': { pb: '16px' },
+              }}
+            >
+              <Box sx={{ flexGrow: 1 }}>
+                <Typography variant="subtitle2" gutterBottom>
+                  Products selected
+                </Typography>
+                <Typography variant="body1">$134.98</Typography>
```

Comment:
```suggestion
                <Typography variant="body1">{activeStep === 2 ? '144.97' ? '$134.98'}</Typography>
```

I think you can try this so the total value changes once you get to the last step of the checkout form.